sdk/maps/Azure.Maps.Routing/README.md

```diff
@@ -211,7 +211,7 @@ RouteMatrixOptions routeMatrixOptions = new RouteMatrixOptions(routeMatrixQuery)
 };
 
 // Invoke an async route matrix request and directly wait for completion
```

Comment:
> an async route

This terminology is potentially confusing.  We generally call these "long-running operations" for public consumption.   The concept of sync/async has meaning in the .NET space that is different than the context in which the sample is using it.